File: 1550-three-consecutive-odds/1550-three-consecutive-odds.java
class Solution {
    public boolean threeConsecutiveOdds(int[] arr) {
        int c=0;
        for(int i=0;i<arr.length;i++){
           if(arr[i]%2!=0){
            if(c==2) return true;
            else c++;
           }else{
            c=0;
           }
        }
       
        return false;
    }
}

File: 1550-three-consecutive-odds/1550-three-consecutive-odds.py
class Solution:
    def threeConsecutiveOdds(self, arr: List[int]) -> bool:
        c=0
        for num in arr:
            if(num%2!=0):
                if(c==2):
                     return True
                c+=1
            else:
                c=0
        return False
        